src/driver.cpp

```diff
@@ -201,4 +208,20 @@ TagStringMap Driver::TagTaxonMap() {
   return m;
 }
 
+void Driver::SetTaxa(const std::map<std::string, uint32_t> taxa) {
+  taxa_ = taxa;
+  taxa_complete_ = true;
+}
+
+void Driver::SortTaxa() {
+  std::map<std::string, uint32_t> taxa;
+  uint32_t new_id = 0;
+  for (const auto &[name, old_id] : taxa_) {
```

Comment:
Is this guaranteed to iterate in alphabetical order?

Reply:
Yes, std::map is guaranteed to iterate over keys in alphabetical order.

https://stackoverflow.com/questions/7648756/is-the-order-of-iterating-through-stdmap-known-and-guaranteed-by-the-standard